event/producer_test.go

```diff
@@ -63,24 +62,24 @@ func TestProducer_SearchDataImport(t *testing.T) {
 		}
 		Convey("When SearchDataImport is called on the event producer", func() {
 
-			err := searchDataImportProducer.SearchDataImport(ctx, searchDataImportTestEvent)
+			err := searchDataImportProducer.SearchDataImport(ctx, expectedSearchDataImportEvent)
 			So(err, ShouldBeNil)
 
 			var avroBytes []byte
+			var testTimeout = time.Second * 5
 			select {
 			case avroBytes = <-pChannels.Output:
-				fmt.Printf("avroBytes: %v\n", avroBytes)
 				log.Event(ctx, "avro byte sent to producer output", log.INFO)
```

Comment:
Sorry just noticed this one - replace log.Event with t.Log?

```suggestion
				t.log("avro byte sent to producer output")
```